integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/ApprovedViewSpecification.groovy

```diff
@@ -90,28 +90,29 @@ class ApprovedViewSpecification extends GebReportingSpec {
   }
 
   def "An old cat can be displayed"() {
+    def threeMonthsAgoPlusOneDay = LocalDate.now().minusMonths(3).plusDays(1).format('yyyy-MM-dd')
+    def threeMonthsAgo = LocalDate.now().minusMonths(3).format('yyyy-MM-dd')
```

Comment:
a clunky work around for the date timebomb! Don't want to spend any more time on it at the moment